themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -11,30 +12,30 @@
 {{- $imgCaption := $thisimg.caption -}}
 {{- $imgCredit := $thisimg.credit -}}
 {{- $prefix := replaceRE "^(http[s]://).+$" "$1" $thisuid -}}
+{{- $align := .Get "align" | default "" -}}
+{{- $inline := .Get "inline" | default "false" -}}
 
 
 <div
-  class="image{{- if (.Get "align") -}}
-    image-{{- .Get "align" -}}
-  {{- else }}
-    {{- if lt $imgWidth 600 -}}image-right{{- end -}}
-  {{- end -}}"
+  class="image{{- if $align }}
+    image-{{ $align }}
+  {{- end }}{{- if eq $inline "true" }}image-inline{{- end }}"
```

Comment:
Let's update this to use a go slice, similar to an array of strings. This will make it read clearer and fix this class spacing issue:

<img width="905" alt="Screenshot 2024-06-17 at 3 01 52 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/faa41224-cce2-4803-8e80-ab1d1ff540d5">
<img width="905" alt="Screenshot 2024-06-17 at 3 01 52 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/faa41224-cce2-4803-8e80-ab1d1ff540d5">



```go
{{ $imageClasses := slice "image" }}
{{ if $align }}
  {{ $imageClasses = $imageClasses | append (printf "image-%s" $align) }}
{{ end }}
{{ if eq $inline "true" }}
  {{ $imageClasses = $imageClasses | append "image-inline" }}
{{ end }}

<div class="{{ delimit $imageClasses " " }}">

```